torch/distributed/launch.py

```diff
@@ -190,6 +191,13 @@ def parse_args():
     parser.add_argument('training_script_args', nargs=REMAINDER)
     return parser.parse_args()
 
+def get_omp_num_threads(local_rank, nproc_per_node):
+    avg_threads_per_process = (int)(multiprocessing.cpu_count() / nproc_per_node)
+    residue_threads = multiprocessing.cpu_count() % nproc_per_node
```

Comment:
question: what's the difference with using `torch.get_num_threads()`?

Reply:
torch is not imported yet in the launch script, I guess torch will set number threads to be omp_num_threads in application, and torch.get_num_threads() can return that